CHANGELOG.md

```diff
@@ -1,6 +1,128 @@
 Change Log
 ==========
 
+Version 3.0.0-alpha01 *(2019-03-14)*
+------------------------------------
+
+ * New: Kotlin Generator
+ 
+   Wire 3 can generate Kotlin data classes. To enable this feature via the command line API, pass in 
+   the `--kotlin_out` parameter that should specify the output directory for the generated `*.kt` 
+   files. 
+   
+   Given the following simple proto:
+   
+   ```proto
+   message Person {
+     required string name = 1;
+     required int32 id = 2;
+     optional string email = 3;
+   }
+   ```
+   
+   the generated Kotlin code will look like the following:
+   
+   ```kotlin
+   data class Person(
+     @field:WireField(tag = 1, adapter = "com.squareup.wire.ProtoAdapter#STRING") val name: String,
+     @field:WireField(tag = 2, adapter = "com.squareup.wire.ProtoAdapter#INT32") val id: Int,
+     @field:WireField(tag = 3, adapter = "com.squareup.wire.ProtoAdapter#STRING") val email: String? =
+         null,
+     val unknownFields: ByteString = ByteString.EMPTY
+   ) : Message<Person, Person.Builder>(ADAPTER, unknownFields) {
+     companion object {
+       @JvmField
+       val ADAPTER: ProtoAdapter<Person> = ... // code omitted for brevity
+   ```
+   
+   Note that, with data classes, the Builder class is redundant and is preserved for compatibility 
```

Comment:
This statement is inaccurate both for Wire and in the general case. Data classes are not a replacement for every usage a builder supports and I'd prefer not to propagate this notion.

Maybe "The `copy` method of data classes replaces most usages of the builder. If your code relies..."

Reply:
Yeah, the statement referenced